release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -61,9 +62,18 @@ public enum ReturnKeyType
 		Next,
 		Done
 	}
-		
+
+	private enum IosContentTypeOverride
```

Comment:
Please add a warning here that these need match the strings in EditBox_iOS.m exactly.

Reply:
Fixed.